README.md

```diff
@@ -1,15 +1,12 @@
-# ForgeGradle 1.2 For Gradle 5 Example Project
+# anatawa12's ForgeGradle 1.2 fork for Gradle 4.4.1+ - example project
 
-This is an example of [fork of ForgeGradle made by anatawa12](https://github.com/anatawa12/ForgeGradle).
-The ForgeGradle is made to support Gradle 4.4.1 or later.
+This is an example mod using the [fork of ForgeGradle-1.2 made by anatawa12](https://github.com/anatawa12/ForgeGradle-1.2).
+This fork supports Gradle 4.4.1 and later. This example project uses Gradle 5.6.4.
 
-## How to use this example
+## How to use this example project
 
-You can download this example form [here](https://github.com/anatawa12/ForgeGradle-example/archive/master.zip).
-You can use this as forge's mdk(in 1.7.10, src).
-This project has inappropriate .gitignore so you should remake .gitignore yourself.
- 
-## How to Use FG1.2 for Gradle 5
+You can download this example project from [here](https://github.com/anatawa12/ForgeGradle-example/archive/master.zip), or use it as a template on Github.
+This project can be used as a replacement for Forge's 1.7.10 MDK.
 
-We changed only a few to use FG1.2 for Gradle 5. 
-[here](https://github.com/anatawa12/ForgeGradle-example/commit/52d6f450a55f93d40365ed0fd2944cea5ca2b4d8)'s the list of changes from forge's mdk(in 1.7.10, src) to use FG1.2 for Gradle5.
+## How to replace the official ForgeGradle in Forge's 1.7.10 MDK with anatawa12's fork
+[Here](https://github.com/anatawa12/ForgeGradle-example/commit/52d6f450a55f93d40365ed0fd2944cea5ca2b4d8)'s a list of changes to Forge's 1.7.10 MDK build script which replaces the official ForgeGradle with anatawa12's fork of ForgeGradle.
```

Comment:
Could you explain by text like README in anatawa12/ForgeGradle-1.2 how to replace the official ForgeGradle in Forge's 1.7.10 MDK with anatawa12's fork? The diff is no longer valid because the diff changes ForgeGradle to 1.2-1.0.0.

Reply:
I've now added more instructions to the README, which should hopefully address this!